src/test/resources/log4j.properties

```diff
@@ -0,0 +1,5 @@
+log4j.rootLogger=INFO, stdout
```

Comment:
Is these java properties configurable when Gerrit service starts? , by appending -Dlog4j.rootLogger=.... to command line or add it to JAVA_OPTS="-Dlog4j.rootLogger=....."

Reply:
The test logger config should only load while testing.

Reply:
ok